coral-spark/src/test/java/com/linkedin/coral/spark/CoralSqlNodeConversionTest.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2022 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.spark;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.calcite.rel.RelNode;
+import org.apache.calcite.rel.rel2sql.SqlImplementor;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.commons.io.FileUtils;
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.apache.hadoop.hive.metastore.api.MetaException;
+import org.apache.hadoop.hive.ql.metadata.HiveException;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.linkedin.coral.spark.containers.SparkRelInfo;
+import com.linkedin.coral.transformers.CoralRelToSqlNodeConverter;
+
+import static org.testng.Assert.*;
+
+
+public class CoralSqlNodeConversionTest {
```

Comment:
Add more tests like testing for `POSEXPLODE`?

Reply:
I had to remove this test class since `Coral SqlNode (1)` is not equivalent to `Coral SqlNode (2)`. The two use some different SqlOperators and hence their unparsing does not generate equivalent SQLs. 
For ex: `Coral SqlNode (1)` uses `HiveExplodeOperator` and `HivePostExplodeOperator` while `Coral SqlNode (2)` uses `CoralSqlUnnestOperator` and these operators differ in unparsing.